diab_encounters/models.py

```
@@ -0,0 +1,18 @@
+class Encounter(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    patient = models.ForeignKey('patients_core.Patient', on_delete=models.CASCADE)
+    occurred_at = models.DateTimeField()
+    subjective = models.TextField(blank=True)
+    objective = models.JSONField(default=dict)
+    assessment = models.JSONField(default=dict)
+    plan = models.JSONField(default=dict)
+    created_by = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.PROTECT)
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
```

Ruff (0.12.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)